RedHat-UX/red-hat-design-system
# fix: Underline links - breadcrumb + navigation-secondary

## What I did

1. Added underlines to links in `rh-breadcrumb` and links in the dropdowns of `rh-navigation-secondary`.

## Testing Instructions

1. Visit the DP for:
  * [Breadcrumbs](https://deploy-preview-1743--red-hat-design-system.netlify.app/elements/breadcrumb/demo/)
  * [Navigation Secondary](https://deploy-preview-1743--red-hat-design-system.netlify.app/elements/navigation-secondary/demo/)
2. Verify the appropriate links are underlined. 

## Notes to Reviewers

1. Currently, links have underlines on hover. If they should be removed, LMK.
2. Is [the changeset](https://github.com/RedHat-UX/red-hat-design-system/pull/1743/files#diff-01f4e7dee70256ba98a1d044f8492c760d22cec0374c64877667fd4602a70849) formatted correctly?

Related to [Jira 1797](https://issues.redhat.com/browse/DXFED-1797).
